man/systemd.link.xml

```diff
@@ -345,9 +345,7 @@
       <varlistentry>
         <term><varname>BitsPerSecond=</varname></term>
         <listitem>
-          <para>The speed to set for the device, the value is rounded
-          down to the nearest Mbps. The usual suffixes K, M, G, are
-          supported and are understood to the base of 1000.</para>
+          <para>The speed to set for the device in Mbps.</para>
```

Comment:
Suffix should continue to be supported. Just convert to the right units internally.